sdk/postgresql/Azure.ResourceManager.PostgreSql/api/Azure.ResourceManager.PostgreSql.netstandard2.0.cs

```diff
@@ -713,29 +828,305 @@ namespace Azure.ResourceManager.PostgreSql.FlexibleServers.Models
+        public static bool operator !=(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.AzureManagedDiskPerformanceTier left, Azure.ResourceManager.PostgreSql.FlexibleServers.Models.AzureManagedDiskPerformanceTier right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class BackupRequestBase
+    {
+        public BackupRequestBase(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerBackupSettings backupSettings) { }
+        public string BackupName { get { throw null; } }
+    }
+    public partial class CapabilityBase
+    {
+        internal CapabilityBase() { }
+        public string Reason { get { throw null; } }
+        public Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexbileServerCapabilityStatus? Status { get { throw null; } }
+    }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExecutionStatus : System.IEquatable<Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ExecutionStatus(string value) { throw null; }
+        public static Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus Cancelled { get { throw null; } }
+        public static Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus Failed { get { throw null; } }
+        public static Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus Running { get { throw null; } }
+        public static Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus Succeeded { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus left, Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus right) { throw null; }
+        public static implicit operator Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus left, Azure.ResourceManager.PostgreSql.FlexibleServers.Models.ExecutionStatus right) { throw null; }
+        public override string ToString() { throw null; }
+    }
+    public partial class FlexibleServersLogFile : Azure.ResourceManager.Models.ResourceData
```

Comment:
Since we have another FlexibleServers in mysql namespace as well, could we rename this to `PostgreSqlFlexibleServersLogFile`?